users/discord/App.js

```diff
@@ -38,6 +38,23 @@ const handleTabNavigation = async (e) => {
 };
 
 const handleUserSelected = (e) => {
+  const userDetailsSection = document.querySelector('.user_details_section');
+  const overlay = document.getElementById('overlay');
+
+  if (window.innerWidth < 600) {
+    overlay.style.display = 'block';
+    userDetailsSection.style.display = 'block';
+    document.body.style.overflow = 'hidden';
+
+    overlay.addEventListener('click', (event) => {
+      if (event.target === overlay) {
+        overlay.style.display = 'none';
+        userDetailsSection.style.display = 'none';
+        document.body.style.overflow = 'auto';
+      }
+    });
+  }
```

Comment:
while your code currently handles the overlay click event well, you might also want to consider allowing users to close the user details section when they click a close button or icon inside the details section itself. This can provide a more intuitive user experience.

Reply:
Users can close the modal by clicking anywhere on the screen except modal itself. Please take a look to the video.

Reply:
okay, cool